slides/snippets/src/main/java/CopyPresentation.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START slides_copy_presentation]
```

Comment:
Did you consider moving the region tags to just include the meaty bits? I understand if developers want to see the whole thing, but would it make sense to include just the most interesting stuff on the page and leave the imports and other setup on GitHub behind a click?

Reply:
IIRC the rest of cloud has been moving towards using nested region tags (<name> for the full sample and <name>_core for the inner part) which gives some flexibility as to which part to show in the docs.

Reply:
Nice, that works.